packages/actions-global/src/actions/show-notification.ts

```diff
@@ -3,15 +3,18 @@ import {
   addNotification,
   TNotificationMetaOptions,
 } from '@commercetools-frontend/notifications';
-import { DOMAINS } from '@commercetools-frontend/constants';
+import {
+  NOTIFICATION_DOMAINS,
```

Comment:
This is just an alias to `DOMAINS` (to be backwards compatible). I find it a bit more clear what "domains" refer to. We can use this internally in app-kit first.

Reply:
I like this new naming a lot!